libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -90,40 +88,6 @@ public TurnContext(ITurnContext turnContext, Activity activity)
         /// <value>The services registered on this context object.</value>
         public TurnContextStateCollection TurnState { get; } = new TurnContextStateCollection();
 
-        /// <summary>
-        /// Gets or sets the locale on this context object.
-        /// </summary>
-        /// <value>The string of locale on this context object.</value>
-        public string Locale
```

Comment:
We cannot remove this, it's public and it would be a breaking change. You could mark it deprecated and recommend the right method to call in the deprecation message (once we close what the right method is)

Reply:
Marked as deprecated.